src/scripts/ui/BuildingUpgradeComponent.tsx

```diff
@@ -59,22 +59,29 @@ export function BuildingUpgradeComponent({ gameState, xy }: IBuildingComponentPr
                </div>
             </div>
             <div className="separator"></div>
-            <div className="row">
-               {levels.map((level, index) => (
-                  <Tippy
-                     content={`${t(L.Upgrade)} x${level}: ${mapOf(
-                        getTotalBuildingCost(building.type, building.level, building.level + levels[index]),
-                        (res, amount) => {
-                           return `${Config.Resource[res].name()} ${formatNumber(amount)}`;
-                        },
-                     ).join(", ")}`}
-                     placement="top"
-                  >
-                     <button key={level} className="f1" onClick={() => upgrade(level)}>
-                        x{level}
-                     </button>
-                  </Tippy>
-               ))}
+            <div className="column">
+               <div className="mb5 text-strong text-small">Upgrade Building</div>
```

Comment:
You might be better off adding the title to the separator.  And you also need to add the text to the translation file (en.ts)

```
               <div className="separator has-title">
                  <div>
                     {t(L.UpgradeBuilding)}
                  </div>
               </div>
```

Reply:
| English | Russian |
| -- | -- |
| ![image](https://github.com/fishpondstudio/CivIdle/assets/40712686/2119b5f6-aa32-49c4-b2ce-7251d37a45e2) | ![image](https://github.com/fishpondstudio/CivIdle/assets/40712686/95e2b573-cedd-445a-91cb-139b173a69d0) |

Added translation as of https://github.com/fishpondstudio/CivIdle/pull/24/commits/9b893958592be33263f3b48e3fdbacd335443ce3